### PR TITLE
Add dd-action-name attributes to MHV Secondary nav

### PR DESCRIPTION
**Pull request by dcloud**
## Summary

- Add `data-dd-action-name` to secondary Nav items

## Related issue(s)

- department-of-veterans-affairs/va.gov-team/issues/81491

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## What areas of the site does it impact?

Tools that use MHV Secondary Nav

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

## Requested Feedback

You can use the Stylus browser extension, similar to how you'd [test for exposed content](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/setup-real-user-monitoring.md#testing-for-exposed-content)

```css
@-moz-document url-prefix("http://localhost:3001"), url-prefix("https://staging.va.gov/") {
:root {
    --hl-color: hsl(120, 77%, 39%);
    --bg-color: hsl(0, 0%, 14%);
}

[data-dd-action-name] {
    outline: 6px double var(--hl-color);
    position: relative;
}

[data-dd-action-name]::after {
    color: var(--hl-color);
    font: caption;
    content: '[dd-action-name]: "' attr(data-dd-action-name) '"';
    background-color: var(--bg-color);
    outline: 6px double var(--hl-color);
    display: block;
    position: absolute;
    left: 0;
    top: 6rem;
    padding: 0.5rem;
    min-width: 180px;
    z-index: 100;
}
}
```

Alternatively, you could run something like `document.querySelectorAll("[data-dd-action-name]").forEach(el => el.style.setProperty('outline', '6px double lime')) ` in the browser console to outline the elements with the prop

### CODE REVIEW

**Comment by dcloud** [comment]
Accidentally removed something I needed while resolving merge conflicts… fix coming soon

File: src/platform/mhv/secondary-nav/components/MhvSecondaryNavItem.jsx [hunk 1]
```diff
@@ -44,7 +45,11 @@ const MhvSecondaryNavItem = ({
 
   return (
     <div key={key} className={itemClass} data-testid="mhv-sec-nav-item">
-      <a href={href} className="vads-u-text-decoration--none">
+      <a
+        href={href}
+        data-dd-action-name={actionName}
```
Comment (radavis) [inline src/platform/mhv/secondary-nav/components/MhvSecondaryNavItem.jsx:50]: Should there be a test to assert this html property?

Comment (dcloud) [reply src/platform/mhv/secondary-nav/components/MhvSecondaryNavItem.jsx:50]: Added a test, but I may be missing something obvious about how to query the attribute using testing-library, so I have `result.getAttribute('data-dd-action-name')` in there, which may or may not be okay… `expect(result).to.have.attribute('data-dd-action-name', actionName)` wasn't working

Comment (radavis) [reply src/platform/mhv/secondary-nav/components/MhvSecondaryNavItem.jsx:50]: That's the way I've done it in [other PRs](https://github.com/department-of-veterans-affairs/vets-website/pull/29583/files#diff-3d59f62b71da7450558604d2c5e339b621022381d30d42ccfe5e9adfb1ea4b76R13-R14).

Comment (radavis) [reply src/platform/mhv/secondary-nav/components/MhvSecondaryNavItem.jsx:50]: Figuring out exactly what you have to say to chai to get what you want is tricky. You might want `.to.have.property()`.

https://www.chaijs.com/guide/styles/

Comment (dcloud) [reply src/platform/mhv/secondary-nav/components/MhvSecondaryNavItem.jsx:50]: I tried `to.have.property()` and it didn't work. Guessing bc an HTMLElement's attributes are not quite the same as props

**Review by radavis — APPROVED** [review]

**Pull request merged**